Gemfile

```diff
@@ -3,7 +3,7 @@
 source 'https://rubygems.org/'
 ruby file: '.ruby-version'
 
-gem 'sequel'
+gem 'sequel', github: 'jeremyevans/sequel'
```

Comment:
To use https://github.com/jeremyevans/sequel/commit/7b748b5c89b1448fb6e1c42037b741421fa32101 (https://github.com/jeremyevans/sequel/discussions/2176)

Reply:
That change is now part of Sequel [5.82.0](https://github.com/jeremyevans/sequel/blob/93987e6b36b1fb5ee0f82342e6a62c5e25070692/CHANGELOG#L9-L22)

Reply:
We now have it since https://github.com/Starkast/wikimum/pull/649